gfx/map/map_object_data/building_locators.txt

```diff
@@ -1759,7 +1759,7 @@
 		}
 		{
 			id=294
-			position={ 3673.000000 0.000000 2232.000000 }
+			position={ 3671.783203 0.000000 2236.604004 }
 			rotation={ -0.000000 -0.000000 -0.000000 1.000000 }
 			scale={ 1.000000 1.000000 1.000000 }
```

Comment:
This island in Darkshore is gone.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/133265288-98de318b-7178-4380-9126-8348273c638f.png)

</details>